subiquity/ui/views/filesystem/guided.py

```diff
@@ -48,111 +57,97 @@
 
 log = logging.getLogger("subiquity.ui.views.filesystem.guided")
 
-
 text = _("""The installer can guide you through partitioning an entire disk \
-either directly or using LVM, or, if you prefer, you can do it manually.
+either directly or using LVM, with or without LUKS, or, if you prefer, you \
+can do it manually.
 
 If you choose to partition an entire disk you will still have a chance to \
 review and modify the results.""")
 
 
-class GuidedFilesystemView(BaseView):
-
-    title = _("Filesystem setup")
-
-    def __init__(self, controller):
-        self.controller = controller
-        direct = ok_btn(
-            _("Use An Entire Disk"), on_press=self.guided, user_arg="direct")
-        lvm = ok_btn(
-            _("Use An Entire Disk And Set Up LVM"), on_press=self.guided,
-            user_arg="lvm")
-        manual = ok_btn(_("Manual"), on_press=self.manual)
-        back = back_btn(_("Back"), on_press=self.cancel)
-        super().__init__(screen(
-            rows=[button_pile([direct, lvm, manual, back]), Text("")],
-            buttons=None,
-            focus_buttons=False,
-            excerpt=text))
-
-    def manual(self, btn):
-        log.debug("GuidedFilesystemView.manual")
-        self.controller.manual()
-
-    def guided(self, btn, method):
-        log.debug("GuidedFilesystemView.guided")
-        self.controller.guided(method)
-
-    def cancel(self, btn=None):
-        self.controller.cancel()
+class GuidedForm(Form):
+
+    # use a proper radio group!!!
```

Comment:
Hum, please, either do it right away, or remove the comment and file a bug instead. Comments like that will just end up forgotten. Same for the padding comments below.